test/verify/check-metrics

```diff
@@ -834,6 +834,70 @@ podman save quay.io/libpod/busybox | sudo -i -u admin podman load
         with b.wait_timeout(300):
             b.wait(lambda: float(b.text("#load-avg .pf-l-flex div:first-child").split()[-1].rstrip(',')) < 10)
 
+        # Files with CPU temperature do not exist, nothing is displayed
+        b.wait_not_present("#current-metrics-card-cpu .temperature-flex")
+
+        # No matching type
+        m.execute("mkdir -p /tmp/hwmon/hwmon0")
+        m.execute("echo BAT0 > /tmp/hwmon/hwmon0/name")
+        m.execute("mount -o bind /tmp /sys/class")
```

Comment:
This is a nondestructive test, so any failure would leave this mounted and thus break the next test. Do this:

    self.addCleanup(m.execute, "umount /sys/class")

which will then run this cleanup on failures as well. Same for creating /tmp/hwmon/.